core/src/main/java/org/ehcache/Ehcache.java

```diff
@@ -781,7 +822,7 @@ public V apply(final K k, final V inCache) {
       }
     });
     try {
-      store.computeIfPresent(key, remappingFunction, REPLACE_FALSE);
+      store.compute(key, remappingFunction, REPLACE_FALSE);
```

Comment:
I would apply YAGNI here. Maybe have a low priority issue around to make sure we ask the question again once we know for sure.